packages/schemas/src/player/gamemodes/woolgames/index.ts

```diff
@@ -0,0 +1,61 @@
+import { APIData } from '@statsify/util';
+import { Color, ColorCode } from '../../../color';
+import { Field } from '../../../metadata';
+import { Progression } from '../../../progression';
+import { WoolWarsStats } from './class';
+import { getExpReq, getFormattedLevel, getLevel } from './util';
+
+export class WoolGames {
```

Comment:
Rename this to `WoolWars`

Reply:
no

Reply:
It doesn't make sense to have the kits in the class if it is going to be called `WoolGames`. If more games are added then I will refactor it back to be `WoolGames`. Until then call it `WoolWars`.